packaging/linux/run_keybase

```diff
@@ -60,7 +60,7 @@ start_systemd() {
   # We don't want to persist this, so don't store it in the env file.
   # This is unset right after start in the unit file so subsequent direct
   # systemctl calls don't have it set.
-  systemctl --user import-environment "KEYBASE_AUTOSTART"
+  systemctl --user set-environment "KEYBASE_AUTOSTART=$KEYBASE_AUTOSTART"
```

Comment:
My mistake on this one. `import-environment` only works when the environment variable is exported, but this variable was just set for this shell script, so we need to use `set-environment` instead.

Reply:
(What this variable does is make it so when Keybase starts up automatically on boot, it is minimized)